Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -13,19 +13,66 @@
 namespace Symfony\CS\Fixer\Contrib;
 
 use Symfony\CS\AbstractFixer;
+use Symfony\CS\ConfigurationException\InvalidFixerConfigurationException;
 use Symfony\CS\Tokenizer\Tokens;
 
 /**
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author Darius Matulionis <darius@matulionis.lt>
  * @author SpacePossum
  */
 class OrderedUseFixer extends AbstractFixer
 {
     const IMPORT_TYPE_CLASS = 1;
+
     const IMPORT_TYPE_CONST = 2;
+
     const IMPORT_TYPE_FUNCTION = 3;
 
+    const SORT_ALPHA = 'alpha';
+
+    const SORT_LENGTH = 'length';
+
+    /**
+     * Sorting type.
+     *
+     * @var string
+     */
+    private static $sortType = self::SORT_ALPHA;
+
+    /**
+     * Array of supported sort types.
+     *
+     * @var string[]
+     */
+    private static $supportedSortTypes = array(self::SORT_ALPHA, self::SORT_LENGTH);
+
+    /**
+     * @param array $sortType
+     */
+    public static function configure(array $sortType = null)
+    {
+        // If no configuration was passed, stick to default.
+        if (null === $sortType) {
+            return;
+        }
+
+        // Configuration should contain only one sort type and can not be empty.
+        if (count($sortType) !== 1) {
+            throw new InvalidFixerConfigurationException('ordered_use', sprintf('Sort type is invalid. Array should contain only one of the parameter: "%s"', implode('", "', self::$supportedSortTypes)));
+        }
+
+        $sortType = array_pop($sortType);
+
+        // Check if passed sort type is supported.
+        if (!is_string($sortType) || !in_array(strtolower($sortType), self::$supportedSortTypes, true)) {
+            throw new InvalidFixerConfigurationException('ordered_use', sprintf('Sort type is invalid. Array should contain only one of the parameter: "%s"', implode('", "', self::$supportedSortTypes)));
+        }
+
+        self::$sortType = strtolower($sortType);
```

Comment:
`strtolower` is not needed here, user should provide proper value, not value that needs to be fixed before use.